proto/trtl/v1/trtl.proto

```diff
@@ -127,8 +128,11 @@ message SyncReply {
 // Options conditions all accesses to trtl, e.g. there are not different structs for
 // Get vs Put options. The semantics of each option depends on the type of request.
 message Options {
-    bool return_meta = 1;   // generally, return the version information for the object in the response
-    bool iter_no_keys = 2;  // do not include keys in an Iter or Batch response, to reduce data transfer load
+    bool return_meta = 1;     // generally, return the version information for the object in the response
+    bool iter_no_keys = 2;    // do not include keys in an Iter or Batch response, to reduce data transfer load
+    bool iter_no_values = 3;  // do not include values in an Iter or Batch response, to reduce data transfer load
+    string page_token = 4;    // specify the page token to fetch the next page of results
+    int32 page_size = 5;      // specify the number of results per page, cannot change between page requests
```

Comment:
I'm not 100% sure about if these should be options or if they should be part of the request. The `return_meta` option is pretty common, but the `iter` options are only useful for `Iter`, `Batch` and `Cursor` (I guess maybe `Sync` too). I'd be happy to discuss this.

Reply:
Note, iter options does not include `Batch` only `Iter` and `Cursor`.